app/helpers/uploads_helper.rb

```diff
@@ -0,0 +1,35 @@
+require 'fileutils'
+require 'audit'
+require 'open3'
+
+module UploadsHelper
+  class Postprocessor
+    def self.process(f)
+      if self.respond_to?(File.extname(f)[1..-1])
```

Comment:
Users get to pick filenames, so this is a security hole.

Better to just have a big case/when.